app/dashboard/views.py

```diff
@@ -1667,7 +1673,7 @@ def change_bounty(request, bounty_id):
 
     result = {}
     for key in keys:
-        result[key] = getattr(bounty, key)
+        result[key] = getattr(bounty, key) or ''
```

Comment:
`getattr(bounty, key, '')`

Reply:
Yay! That didn't work for me `getattr(obj, key /* = None */, '') == None` 😒

Reply:
@pinkiebell Makes me wonder why something like `key` being a string would be nullable 😭

Reply:
Nah, The key is fine, the value is `None`. That's why I'm confused with `getattr`